awx/main/conf.py

```diff
@@ -408,6 +408,18 @@
     unit=_('seconds'),
 )
 
+register(
+    'DEFAULT_JOB_IDLE_TIMEOUT',
+    field_class=fields.IntegerField,
+    min_value=0,
+    default=600,
```

Comment:
Is this also the default value used by runner for `idle_timeout`?

If we add this setting using a `default` of `600`, will impact the existing behavior of preexisting job templates after upgrading?